### PR TITLE
docs v0.7.0 denote required fields

### LINKED ISSUE #1809: Docs should note that label is required when creating a disk without a filesystem

### LINKED ISSUE #1931: collapse docs examples by default

### PR DESCRIPTION
**Pull request by mparke**
closes #1809 
closes #1931 

screen shot 2017-06-06 at 9 40 21 am
screen shot 2017-06-06 at 9 40 36 am
screen shot 2017-06-06 at 9 40 40 am

### CODE REVIEW

**Comment by eatonphil** [comment]
Nice.

File: docs/src/components/tables/Cells/ParamFieldCell.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    subLabel = (
+      <div className="Label Label-required">
+        required
+      </div>
+    );
+  } else {
+    subLabel = (
+      <div className="Label Label-optional">
+        optional
+      </div>
```
Comment (eatonphil) [inline docs/src/components/tables/Cells/ParamFieldCell.js]: Why not `<small className="text-muted">optional</small>`?

File: docs/src/components/tables/Cells/ParamFieldCell.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    );
+  }
+
+  return (
+    <td className={`TableCell FieldCell`}>
```
Comment (eatonphil) [inline docs/src/components/tables/Cells/ParamFieldCell.js]: No need for the `{``}` here. `className="TableCell FieldCell"` would suffice.

File: docs/scss/components/MethodResponseExample.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
+.MethodResponseExample {
+    &-header {
+        border-bottom: $border;
```
Comment (eatonphil) [inline docs/scss/components/MethodResponseExample.scss]: Could we drop this?

File: docs/scss/components/MethodResponseExample.scss [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        padding: 10px;
+        position: relative;
```
Comment (eatonphil) [inline docs/scss/components/MethodResponseExample.scss]: Is this needed?

File: docs/scss/components/Tables/cells/FieldCell.scss [hunk 5]
```diff
@@ -0,0 +1,7 @@
+.FieldCell {
+    height: 60px;
```
Comment (eatonphil) [inline docs/scss/components/Tables/cells/FieldCell.scss]: Seems a little scary this is set like that. Can it not just be determined by default td height or default td padding?

File: docs/src/components/MethodResponseExample.js [hunk 6]
```diff
@@ -1,15 +1,39 @@
...
+
+  onClick() {
+    this.setState({ collapsed: !this.state.collapsed });
+  }
+
+  render() {
+    const { resource } = this.props;
+    const { collapsed } = this.state;
+
+    const caretClass = collapsed ? 'fa-caret-right' : 'fa-caret-down';
```
Comment (eatonphil) [inline docs/src/components/MethodResponseExample.js]: Could we replace this with Show / Hide or the like?

File: docs/scss/components/Tables/cells/FieldCell.scss (hunk 5 above)
Comment (mparke) [inline docs/scss/components/Tables/cells/FieldCell.scss]: what is scary about it? creates a minimum height for these cells

File: docs/scss/components/MethodResponseExample.scss (hunk 3 above)
Comment (mparke) [inline docs/scss/components/MethodResponseExample.scss]: looks out of place without something here. open to ideas

File: docs/scss/components/Tables/cells/FieldCell.scss (hunk 5 above)
Comment (mparke) [inline docs/scss/components/Tables/cells/FieldCell.scss]: fixed, was not needed, thanks

File: docs/scss/components/MethodResponseExample.scss (hunk 4 above)
Comment (mparke) [inline docs/scss/components/MethodResponseExample.scss]: fixed, was no longer needed

**Comment by mparke** [comment]
requires https://github.com/linode/manager/pull/2023

**Comment by eatonphil** [comment]
Aside from the Example show/hide not being super clear lgtm.

**Comment by mparke** [comment]
show/hide text added, right aligned:

screen shot 2017-06-07 at 12 22 25 pm
screen shot 2017-06-07 at 12 22 28 pm

Sub headings unbolded but increased in size, margins increased slightly:

screen shot 2017-06-07 at 12 22 37 pm

**Pull request merged**